packages/babel-plugin-transform-es2015-template-literals/README.md

```diff
@@ -75,7 +75,7 @@ In loose mode, tagged template literal objects aren't frozen.
 
 `boolean`, defaults to `false`.
 
-This option wraps all template literal expressions with `String`. See [babel/babel#1065](https://github.com/babel/babel/issues/1065) for more info.
```

Comment:
Why did we remove the link? We should add a reason for why this is necessary or a good idea so people understand why they would use this option